// Copyright 2023 Ant Group Co., Ltd.
// 
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
// 
//   http://www.apache.org/licenses/LICENSE-2.0
// 
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto3";

package envoy.extensions.filters.http.kuscia_gress.v3;
option go_package = "gitlab.alipay-inc.com/secretflow/kuscia-envoy/kuscia/api/filters/http/kuscia_gress/v3";

message Gress {
  string instance = 1;
  string self_namespace = 2;
  bool rewrite_host = 3;
  bool add_origin_source = 4;
  int32 max_logging_body_size_per_reqeuest = 5;
}

